admin/anspress-admin.php

```diff
@@ -720,7 +720,9 @@ private static function check_pages_exists() {
 
 			$pages_in = array();
 			foreach ( $pages_slug as $slug ) {
-				$pages_in[] = $opt[ $slug ];
+				if ( $opt[ $slug ] ) {
```

Comment:
@mebishalnapit Please use `empty` function as this will still log error when its not set and we don't want empty items.

Reply:
@rahularyan Updated with check for not empty.